.yamato/gym-interface-test.yml

```diff
@@ -0,0 +1,31 @@
+test_editors:
+  - version: 2019.3
+---
+{% for editor in test_editors %}
+test_gym_interface_{{ editor.version }}:
+  name: Test Mac Gym Interface {{ editor.version }}
+  agent:
+    type: Unity::VM::osx
+    image: ml-agents/ml-agents-bokken-mac:0.1.4-492264
+    flavor: b1.small
+  variables:
+    UNITY_VERSION: {{ editor.version }}
+  commands:
+    - pip install pyyaml
+    - python -u -m ml-agents.tests.yamato.run_script_in_venv ml-agents/tests/yamato/scripts/run_gym.py
```

Comment:
I'm thinking about splitting this up into two commands - one to setup the venv (which could take options for additional python libraries) and another to just run the script on its own. Thoughts?

Reply:
Splitting would make it easier to read no ?

Reply:
seems logical

Reply:
Done